examples/multi-plugins/README.md

```diff
@@ -1,6 +1,42 @@
+<!--
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### How to run the example
+
+This example includes configuring and starting influxdb (a time series database) and grafana (a metrics dashboard).
+
 The example:
+  - Starts containers for Influxdb and Grafana (using Docker)
+  - Starts snapd
+  - Gets, loads and starts snap plugins
+  - Creates and starts a task that collects from psutil, cpu, and cgroups then publishes to InfluxDB
+  
+Start the example by running `run-multi.sh`
+
+### Requirements
+- docker
+- docker-compose 
+- netcat 
+- SNAP_PATH env var set to the snap build directory
```

Comment:
this demo you're using as a baseline is working on a revision. That shouldn't be an issue.